Tests/Crypto/BIP32/BIP32KeypairDerivationTests.swift

```diff
@@ -0,0 +1,96 @@
+import XCTest
+import FearlessUtils
+import IrohaCrypto
+
+class BIP32KeypairDeriviationTests: XCTestCase {
+
+    func testBIP32DerivationPath() throws {
+        try performTest(filename: "BIP32HDKD", keypairFactory: BIP32KeypairFactory(), useMiniSeed: false)
+    }
+
+    func testBIP32DerivationPathStandardData() throws {
+        try performSeedReadyTest(filename: "BIP32HDKDStandard", keypairFactory: BIP32KeypairFactory())
+    }
+
+    private func performTest(filename: String, keypairFactory: KeypairFactoryProtocol, useMiniSeed: Bool = true) throws {
```

Comment:
Why do we need `useMiniSeed` option if it is not used

Reply:
Here we don't need it, yes